src/command/cmd_defs.c

```diff
@@ -2833,3 +2833,80 @@ command_docgen(void)
     printf("\nProcessed %d commands.\n\n", g_list_length(cmds));
     g_list_free(cmds);
 }
+
+void
+command_mangen(void)
+{
+    GList* cmds = NULL;
+
+    for (unsigned int i = 0; i < ARRAY_SIZE(command_defs); i++) {
+        Command* pcmd = command_defs + i;
+        cmds = g_list_insert_sorted(cmds, pcmd, (GCompareFunc)_cmp_command);
+    }
+
+    mkdir_recursive("docs");
+
+    char* header = NULL;
+    GDateTime *now = g_date_time_new_now_local();
+    gchar *date = g_date_time_format(now, "%F");
+    if (asprintf(&header, ".TH man 1 \"%s\" \""PACKAGE_VERSION"\" \"Profanity XMPP client\"\n", date) == -1) {
+        // TODO: error
```

Comment:
I guess you could add a simple `printf` here? :)

Reply:
True. Do you think I should? PACKAGE_VERSION probably wont get to 1231.123311... :D